angular/src/lib/icon/icon.component.ts

```diff
@@ -18,24 +19,15 @@ const colors = require('@collab-ui/core/data/colors.json');
 @Component({
   selector: 'cui-icon',
   template: `
-  <i
-  *ngIf="!isClickable"
-  class={{classes}}
-  [style.color]="getColor()"
-  aria-labelledby="getAriaLabelledBy()"
-  aria-label="getAriaLabel()">
-</i>
-
-<button
-  *ngIf="isClickable"
-  class={{buttonClasses}}
-  aria-labelledby="getAriaLabelledBy()"
-  aria-label="getAriaLabel()">
-  <i  class={{classes}}
-      [style.color]="getColor()">
-  </i>
-</button>
-`,
+    <button
+      *ngIf="isClickable"
+      class={{buttonClasses}}
+      [attr.aria-labelledby]="getAriaLabelledBy()"
+      [attr.aria-label]="ariaLabel"
+    >
+      <i class={{classes}} [style.color]="getColor()"></i>
```

Comment:
@wzpo I don't understand how anything would get rendered in the case that the icon component is not clickable.

Reply:
@elenagiere Each of the angular component has host tag. I bind classes and other props to the host element instead of the child `<i>`.

The icon is rendered like this before:
`<cui-icon>
  <i class="cui-icon icon icon-cancel_16"></i>
</cui-icon>`

after:
`<cui-icon class="cui-icon icon icon-cancel_16"></cui-icon>`

React component:
`<i class="cui-icon icon icon-cancel_16"/>`

There's no different between `cui-icon` and `i` in this component.